### ISSUE TITLE
[libgpl] - Save email as event is broken

### ISSUE THREAD

**Issue by GoogleCodeExporter** [roundcubevnz/myroundcube#830]
```
https://forum.myroundcube.com/topic/408-calendar-save-email-as-event-not-working
```

Original issue reported on code.google.com by `myroundc...@gmail.com` on 3 Jun 2015 at 7:12

**Comment by GoogleCodeExporter** [comment]
```
Fixed in libgpl 1.1.52
```

Original comment by `myroundc...@gmail.com` on 30 Jun 2015 at 12:34
- Changed title: **[libgpl] - Save email as event is broken**

**Comment by GoogleCodeExporter** [comment]
Original comment by `myroundc...@gmail.com` on 30 Jun 2015 at 12:34
- Changed state: **Fixed**
